lib/parsers/csv_layout.rb

```diff
@@ -41,16 +41,29 @@ def builder(barcode)
     end
 
     def valid_location?(location)
-      !!location.match(/[A-H]\d\d/)
+      !!location.match(/^[A-H]\d{1,2}$/)
     end
 
     def valid_fluidx_barcode?(barcode)
       barcode.start_with?('F')
     end
 
     def no_read_barcode?(barcode)
-      barcode.start_with?('No Read')
-    end    
+      barcode.downcase.start_with?('no read')
+    end
+
+    def convert_to_location(str)
+      if valid_location?(str)
+        matches = str.match(/^([A-H])(\d{1,2})$/)
+        num = matches[2]
+        if (num.length==1)
```

Comment:
Rubocop is not going to like the lake of space between operators.. `"0" + num...` Does this project have rubocop?